e2e-test/tsps_e2e_test.py

```diff
@@ -126,6 +129,39 @@ def poll_for_imputation_job(tsps_url, job_id, token):
     raise Exception(f"tsps pipeline did not complete in 25 minutes")
 
 
+# retrieve a specified row from a specified table in a specified workspace's WDS
+def retrieve_wds_row(wds_url, workspace_id, record_type, record_id, azure_token):
+    version="v0.2"
+    api_client = wds_client.ApiClient(header_name='Authorization', header_value="Bearer " + azure_token)
```

Comment:
any reason why we use the wds client here but in the `set_up_control_workspace` script we make http requests when talking to wds?

Reply:
just that I wrote the http request function back when we were trying to avoid using wds_client. I'll update it now to use the existing function in `helper.py`